paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -218,18 +228,21 @@ class PaparazziPlugin : Plugin<Project> {
       operatingSystem.isWindows -> "win"
       else -> "linux"
     }
-    unzipConfiguration.dependencies.add(
+    nativePlatformConfiguration.dependencies.add(
       dependencies.create("app.cash.paparazzi:layoutlib-native-$nativeLibraryArtifactId:$NATIVE_LIB_VERSION")
     )
     dependencies.registerTransform(UnzipTransform::class.java) { transform ->
-      transform.from.attribute(ArtifactAttributes.ARTIFACT_FORMAT, ArtifactTypeDefinition.JAR_TYPE)
+      transform.from.attribute(
+        ArtifactTypeDefinition.ARTIFACT_TYPE_ATTRIBUTE,
+        ArtifactTypeDefinition.JAR_TYPE
+      )
       transform.to.attribute(
-        ArtifactAttributes.ARTIFACT_FORMAT,
+        ArtifactTypeDefinition.ARTIFACT_TYPE_ATTRIBUTE,
```

Comment:
Note, this bumps your plugins minimum version of Gradle to 7.3, just so you are aware.

https://cs.android.com/android-studio/gradle/+/master:subprojects/core-api/src/main/java/org/gradle/api/artifacts/type/ArtifactTypeDefinition.java;l=39

In fact, I'd suggest setting up a test to use different versions of Gradle you wish to support.

Reply:
reverted back to original with comment to avoid burdening consumers who might be able to upgrade yet